apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.test.js

```diff
@@ -1585,6 +1592,91 @@ describe('RegisterMultiPaymentService', () => {
             })
         })
 
+        test('Invoices for different organizations are supported', async () => {
+            const [o10n1] = await createTestOrganization(adminClient)
+            const [o10n2] = await createTestOrganization(adminClient)
+            const [property1] = await createTestProperty(adminClient, o10n1)
+            const [property2] = await createTestProperty(adminClient, o10n2)
+
+            const residentClient = await makeClientWithResidentUser()
+            const unitType = FLAT_UNIT_TYPE
+            const unitName = faker.lorem.word()
+
+            const [resident] = await registerResidentByTestClient(
+                residentClient,
+                {
+                    address: property1.address,
+                    addressMeta: property1.addressMeta,
+                    unitType,
+                    unitName,
+                })
+
+            const staffClient = await makeClientWithStaffUser()
+            const [role] = await createTestOrganizationEmployeeRole(adminClient, o10n1, {
+                canManageInvoices: true,
+                canManageContacts: true,
+            })
+            await createTestOrganizationEmployee(adminClient, o10n1, staffClient.user, role)
+
+            const implicitFeePercent = '5'
+            await createTestAcquiringIntegrationContext(adminClient, o10n2, dummyAcquiringIntegration, {
+                invoiceStatus: CONTEXT_FINISHED_STATUS,
+                invoiceRecipient: createTestRecipient(),
+                invoiceImplicitFeeDistributionSchema: [{
+                    recipient: 'organization',
+                    percent: implicitFeePercent,
+                }],
+            })
+
+            const [marketCategory] = await createTestMarketCategory(adminClient)
```

Comment:
Looks, like you can add MarketPlaceMixin for faster creation of MarketPlaceItem with price, scope, category ...
this MixIn can also update AcquiringContext 
Not in this PR, only to think about it